extensions/amp-form/0.1/amp-form.js

```diff
@@ -209,7 +209,7 @@ export class AmpForm {
         this.actions_.trigger(this.form_, 'submit-error', null);
         this.setState_(FormState_.SUBMIT_ERROR);
         this.renderTemplate_(error.responseJson || {});
-        rethrowAsync('Form submission failed:', error);
+        rethrowAsync(user().createError('Form submission failed:', error));
```

Comment:
@mkhatib could answer this better, but I think any error here is user related, as the errors come from `#fetchJSON`. It's possible the error could be from a sync exception in the `#then` block directly before this `#catch` (which is definitely a dev error), but we could avoid that by moving this block to be the second parameter to the `#then` above.